packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -66,135 +47,171 @@ export default function ScheduleInputs(props: Props) {
-                      )}
-                      onChange={(e) => {
-                        setDateErrors({ [1]: "" });
-                        if (
-                          new Date(e.target.value) <
-                          new Date(rules[0].timestamp)
-                        ) {
-                          setDateErrors({
-                            [1]: "Date must be greater than the previous rule date.",
-                          });
-                          return;
-                        }
-                        onChange(e.target.value, "timestamp", 1);
-                      }}
-                      name="timestamp"
-                    />
-                    <span className="pl-2">
-                      ({formatTimeZone(new Date(), "z")})
-                    </span>
-                  </div>
-                  {dateErrors[1] && (
-                    <div className="ml-2 alert alert-danger mb-0">
-                      {dateErrors[1]}
+            </li>
+            <li className={styles.listitem}>
+              <div className="row align-items-center">
+                <span className="ml-2 mb-2">Disable rule </span>
+                <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
+                  <SelectField
+                    name="date-operator"
+                    value={rules[1].timestamp === null ? "" : "timestamp"}
+                    options={[
+                      { label: "manually", value: "" },
+                      {
+                        label: "at a specific date and time",
+                        value: "timestamp",
+                      },
+                    ]}
+                    onChange={(value) => {
+                      if (value) {
+                        onChange(
+                          format(new Date(), "yyyy-MM-dd'T'HH:mm"),
+                          "timestamp",
+                          1
+                        );
+                      } else {
+                        onChange(null, "timestamp", 1);
+                      }
+                    }}
+                  />
+                </div>
+                {rules[1].timestamp !== null && (
+                  <>
+                    <div className="w-auto mb-2 p-2">
+                      <span className="mb-2">ON</span>
                     </div>
-                  )}
-                </>
-              )}
-            </div>
-          </li>
-        </ul>
-      </div>
+                    <div className="col-sm-12 col-md mb-2 d-flex align-items-center">
+                      <Field
+                        type="datetime-local"
+                        className={clsx(dateErrors[1] && styles.error)}
+                        value={format(
+                          new Date(rules[1].timestamp),
+                          "yyyy-MM-dd'T'HH:mm"
+                        )}
+                        onChange={(e) => {
+                          setDateErrors({ [1]: "" });
```

Comment:
Why is this a tuple? I don't ever see dateErrors[0] being set. And even if it were set, it seems we overwrite it instantly during assignment. That is, if `dateErrors === { 0: "first date is wrong" }`, when we `setDateErrors({ [1] : "" })`, it will overwrite the 0-th error. To not overwrite, we'd need something like
```
setDateErrors({ 
  ...dateErrors,
  [1] : ""
})
```

Reply:
This is an artifact, now - I originally set this file up thinking we'd support multiple start/stop dates, but after getting feedback, just hardcoded the form to only support two dates (1 start, 1 stop).

I've just updated the dateErrors to be a simple string, since the only time we're setting an error is if the stop date is before the start date.

Re: the overwriting - the pattern that I went with was, onChange, we always reset the error, and then run the same check to see if the stop date is before the start date. If there is a different pattern you suggest, let me know.